### ISSUE TITLE
Unnecessary double quotes doesn't detect empty strings

### ISSUE THREAD

**Issue by MazeChaZer** [kalessil/phpinspectionsea#180]
```php
<?php
$a = "";
```

These double quotes can be safely replaced with single quotes.

**Comment by kalessil** [comment]
Ouch, my bad - will be fixed in the next release.
